modules/recommended-etcd-practices.adoc

```diff
@@ -19,3 +19,32 @@ only accepts key reads and deletes. Some of the key metrics to monitor are
 database usage after a history compaction, and
 `etcd_debugging_mvcc_db_total_size_in_bytes` which shows the database size
 including free space waiting for defragmentation.
+
+Etcd writes data to disk, so its performance strongly depends on disk performance. Etcd 
+persists proposals on disk, slow disks and disk activity from other processes may cause 
+long fsync latencies causing etcd to miss heartbeats leading to request timeouts and temporary 
+leader loss. It is highly recommended to run etcd on machines backed by SSD disks with low 
+latency and high throughput. Some of the key metrics to monitor on a deployed OpenShift cluster 
+are p99 of etcd disk write ahead log duration and the number of etcd leader changes.
+Prometheus can be leveraged to keep track of the these metrics, `etcd_disk_wal_fsync_duration_seconds_bucket`
```

Comment:
Maybe finish the sentence after "Prometheus can be leveraged to keep track of the these metrics." and start a new one with `etcd_disk_wal_fsync_duration_seconds_bucket` ......